tests/min_mapped_reads.nf.test

```diff
@@ -13,797 +13,17 @@ nextflow_pipeline {
         }
 
         then {
+            def file_complement    = UTILS.getAllFilesFromDir(params.outdir, null, UTILS.exclusionRegexesForUnstableFileNames)
```

Comment:
```suggestion
            def file_complement    = UTILS.getAllFilesFromDir(params.outdir, null, UTILS.exclusionRegexesForUnstableFileNames, true)
```
Getting relative paths here will make more sense when there's duplicated files between directories.

Reply:
yeah, so this is not working